pkg/apis/kubeone/validation/validation.go

```diff
@@ -352,17 +352,45 @@ func ValidateContainerRuntimeConfig(cr kubeoneapi.ContainerRuntimeConfig, versio
 func ValidateClusterNetworkConfig(c kubeoneapi.ClusterNetworkConfig, fldPath *field.Path) field.ErrorList {
```

Comment:
Please add some unit tests for the newly-added validation cases.

Reply:
Will do as soon as we finish the api discussion. :+1:

Reply:
done.